docs/reference/api.md

```diff
@@ -0,0 +1,29 @@
+# API
+
+<link rel="stylesheet" type="text/css" href="https://unpkg.com/swagger-ui-dist@5.11.0/swagger-ui.css" ></link>
```

Comment:
This is probably fine for now. But just for reference, the LXD docs import this from the repo into `_static` at build time: https://github.com/canonical/lxd/blob/510c0117cb20591f92126e0b8d3bb583118cd93e/doc/custom_conf.py#L291